.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,76 @@
+name: "\U0001F41B Bug Report"
+description: Submit a bug report to help us improve optimum
+labels: [ "bug" ]
+body:
+  - type: textarea
+    id: system-info
+    attributes:
+      label: System Info
+      description: Please share your system info with us. Among others, you can run the command `transformers-cli env` and copy-paste its output below.
```

I don't think we have a CLI for `optimum` yet, but we should! Can you please open a separate issue to track this?

```suggestion
      description: Please share your system info with us.
```

---

Will do, thanks for the review!